gamemodes/terrortown/gamemode/shared/lang/fr.lua

```diff
@@ -7,36 +7,36 @@ local L = LANG.CreateLanguage("fr")
 -- https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/terrortown/gamemode/lang/french.lua
 L.__alias = "français"
 
-L.lang_name = "Francais (French)"
+L.lang_name = "Français (French)"
 
 --- General text used in various places
-L.traitor = "Traitre"
-L.detective = "Détective"
-L.innocent = "Innocent"
+L.traitor = "TRAITRE"
+L.detective = "DÉTECTIVE"
+L.innocent = "INNOCENT"
```

Comment:
Is there a reason for these uppercase letters?

Reply:
yes cause of détective É in some menu he look like the same than the screenshot i have send before so i have put in uppercase Traitre and Innocent too

Reply:
ugh, I hate how GMod handles this. Same thing happens in German as well. I don't know what would be the best solution.